snowplow/iglu-client-embedded/schemas/com.metabase/account/jsonschema/1-0-1

```diff
@@ -15,7 +15,6 @@
       "enum": [
         "new_user_created",
         "new_instance_created",
-        "db_connection_banner_click"
```

Comment:
It's likely not super important, but in general we don't want to edit existing schemas like this. I believe the correct way to handle this would be to create schema 1-0-2 that is almost the same as 1-0-1 but removes this value from the `enum`